### ISSUE TITLE
How to run package installed from pip?

### ISSUE THREAD

**Issue by comfortablynick** [ColdGrub1384/Pyto#94]
Hello,

Thanks for the app -- I'm glad Pythonista has some competition! 

I am writing an [image manipulation tool](https://pypi.org/project/pyimgtool/) to help with resizing/watermarking images which I would like to eventually be able to use on iOS through Pyto+Shortcuts. I installed the package using the PyPI page in Pyto settings and tapping "Install." It seems to be there, but I can't find a way to run it.

On the desktop, simply running `pip install pyimgtool` works fine. Is there something I'm missing?

**Comment by comfortablynick** [comment]
Yes, that's the process I used to install the package. Now how to I run it as an executable? Everything I've tried doesn't work.

**Comment by cclauss** [comment]
1. Open the editor.
2. At lower left click ...
3. At upper right hit +
4. Hack in the Arguments textbox and hit Ok.

**Comment by comfortablynick** [comment]
That wasn't working either, but I figured it out. It looks like pyto runs everything in the `python -m PACKAGE` format, and that works fine for single-file packages. If a package has multiple files, it _must_ have a `__main__.py` so that python knows what file to execute. Apparently, it does not use the `entry_points` list from `setup.py`.

Once I renamed `main.py` to `__main__.py`, it was able to run inside pyto. Good to know!
